setup.py

```diff
@@ -1,690 +1,703 @@
-"""This file installs Pyfhel in your Python3 distribution.

```

Comment:
Once again, the entire file has been modified, thus one cannot follow the changes that took place. Make sure your commit includes only the lines that you have modified!